CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to
 
 ## [Unreleased]
 
+## [0.1.11] - 2024-12-06
+
+### Added
+
+- A new operation to add foreign key fields to an existing table has been
+  added: `SaferAddFieldForeignKey`
```

Comment:
I forgot to add an entry to the change log in the previous PR that added this op, so I'm doing this here instead 🤦🏽